test/unit/util/test_translator.py

```diff
@@ -195,3 +211,15 @@ def test_translate(default_translator, mock_box_session):
     # It should not modify the original
     assert isinstance(response_object['entries'][0], dict)
     assert isinstance(response_object['entries'][1], dict)
+
+
+def test_translator_translates_metadata_template(default_translator, mock_box_session, metadata_template_response):
+    metadata_template = default_translator.translate(mock_box_session, metadata_template_response)
+
+    if metadata_template_response.get('id'):
+        assert metadata_template.id == metadata_template_response['id']
+        assert metadata_template.object_id == metadata_template_response['id']
+
+    if metadata_template_response.get('scope') and metadata_template_response.get('templateKey'):
+        assert metadata_template.scope == metadata_template_response['scope']
+        assert metadata_template.template_key == metadata_template_response['templateKey']
```

Comment:
I think you have this in a different test too, but just for completeness, can we assert that `metadata_template.object_id is None` if `metadata_template_response.get('id') is None`